app/helpers/request.js

```diff
@@ -0,0 +1,19 @@
+const axios = require('axios');
+const { errors } = require('../errors');
+
+const buildRequest = ({ url, path = '', method, headers, data, params }) => ({
+  method,
+  url: `${url}${path}`,
+  headers,
+  data,
+  params
+});
+
+module.exports = options =>
+  axios(buildRequest(options))
+    .then(info => info.data)
+    .catch(error => {
+      const { statusText } = error.response;
+      const message = error.response.data.error;
+      throw errors.axiosError(message, statusText);
```

Comment:
The second param is unnecessary, check this file `app/errors/axios.js` please.

Reply:
Sorry for hear that, already fixed it! My Stupid 'layer 8'.